Documentation/xcassets/swift2.md

```diff
@@ -21,6 +21,8 @@ You can customize some elements of this template by overriding the following par
 | Parameter Name | Default Value | Description |
 | -------------- | ------------- | ----------- |
 | `enumName` | `Asset` | Allows you to change the name of the generated `enum` containing all assets. |
+| `colorTypeName` | `ImageAsset` | Allows you to change the name of the generated color container type. |
```

Comment:
`colorTypeName` defaults to… `ImageAsset`? Shouldn't it be `s/ImageAsset/ColorAsset/`?